pcsx2-qt/MainWindow.cpp

```diff
@@ -1962,6 +2021,12 @@ void MainWindow::onVMStarted()
 	updateWindowTitle();
 	updateStatusBarWidgetVisibility();
 	updateInputRecordingActions(true);
+	if (s_record_on_start)
+	{
+		m_ui.actionVideoCapture->setEnabled(true);
```

Comment:
You've removed the only occurrence of this action being disabled. The setEnabled call can be omitted here, as it will always be enabled.